src/zenml/cli/__init__.py

```diff
@@ -52,16 +52,8 @@
 Beginning a Project
 -------------------
 
-When you begin a project, you should first initialize your base project
-directory as a Git repository. To do this, type:
-
-.. code:: bash
-
-   git init
-
-Once your directory is setup with a (usually hidden) ``.git`` folder,
-initialize the directory with ZenML’s own config and resource management
-tools. Type:
+When you begin a project, initialize the directory with ZenML’s
+own config and resource management tools. Type:
```

Comment:
Isnt this missing something?

Reply:
I have changed it a bit more to reflect the current situation.